proto/api/api.proto

```diff
@@ -279,7 +279,8 @@ message EstimateGasForActionResponse {
 message ReadStateRequest {
   bytes protocolID = 1;
   bytes methodName = 2;
-  repeated bytes arguments = 3;
+  uint64 epochHeight = 3;
```

Comment:
should this be `epochHeight` or just `height`?

Reply:
And also if we have this field to be optional, like if the field is not present, get tip height data? then this could be a little bit weird, since empty is 0 here. Can we accept that, 0 is get tip height data? @CoderZhi  if we don't want that be like this, we may need a different endpoint

Reply:
we can use ```height```, but for current poll protocol use case, in readState handler in api.go, we need to compare based on epochstartheight,  right? (derive from height)

Reply:
@yutongp then how about using bytes for epochHeight?

Reply:
if we plan to use something not uint64, string is better than bytes